HelpSource/Reference/Control-Structures.schelp

```diff
@@ -11,63 +11,69 @@ section:: Basic Control Structures
 method:: if
 
 Conditional execution is implemented via the code::if:: message. The code::if:: message is sent to an expression which must return a link::Classes/Boolean:: value.
+
 In addition it takes two arguments: a function to execute if the expression is true and another optional function to execute if the expression is false. The code::if:: message returns the value of the function which is executed. If the falseFunc is not present and the expression is false then the result of the if message is nil.
 
 discussion::
 Syntax
 code::
-if (expr, trueFunc, falseFunc);
-::
---or--
-code::
+if(expr) { true body } { false body };
+
+// alternate
+if(expr, trueFunc, falseFunc);
+
+// possible but harder to read, so, not preferred
 expr.if (trueFunc, falseFunc);
 ::
 
 Examples
 code::
-if ( [false, true].choose,				// Boolean expression (chooses one at random)
-	{ "expression was true".postln },	// true function
-	{ "expression was false".postln }	// false function
+if([false, true].choose) {
+	"expression was true".postln  // true function
+} {
+	"expression was false".postln  // false function
+};
 )
 
 (
 var a = 1, z;
-z = if (a < 5, { 100 },{ 200 });
+z = if(a < 5) { 100 } { 200 };
 z.postln;
 )
 
 (
 var x;
-if (x.isNil, { x = 99 });
+if(x.isNil) { x = 99 };
 x.postln;
 )
 ::
 
 
-
 method:: while
 
-The while message implements conditional execution of a loop. If the testFunc answers true when evaluated, then the bodyFunc is evaluated and the process is repeated. Once the testFunc returns false, the loop terminates.
+The while message implements conditional execution of a loop. If the testFunc answers true when evaluated, then the loopFunc is evaluated and the process is repeated. Once the testFunc returns false, the loop terminates.
 
 discussion::
 Syntax
 code::
-while ( testFunc, bodyFunc );
-::
---or--
-code::
+while { testFunc body } { loopFunc body };
+
+// alternate
+while (testFunc, bodyFunc);
```

Comment:
I think this should also be loopFunc instead of bodyFunc? 
Thinking about beginner users, how about we do `while ({ testFunc body }, { loopFunc body })` for similarity with the previous example?

Reply:
Agreed -- better to be consistent. Done.